source/Halibut.TestUtils.CompatBinary.Base/ServiceFactoryFactory.cs

```diff
@@ -29,9 +29,13 @@ public static DelegateServiceFactory CreateServiceFactory()
         public static DelegateServiceFactory CreateProxyingServicesServiceFactory(HalibutRuntime clientWhichTalksToLatestHalibut, ServiceEndPoint realServiceEndpoint)
         {
             var forwardingEchoService = clientWhichTalksToLatestHalibut.CreateClient<IEchoService>(realServiceEndpoint);
+            var forwardingCachingService = clientWhichTalksToLatestHalibut.CreateClient<ICachingService>(realServiceEndpoint);
+            var forwardingMultipleParametersTestService = clientWhichTalksToLatestHalibut.CreateClient<IMultipleParametersTestService>(realServiceEndpoint);
             var services = new DelegateServiceFactory();
             services.Register<IEchoService>(() => new DelegateEchoService(forwardingEchoService));
+            services.Register<ICachingService>(() => new DelegateCachingService(forwardingCachingService));
```

Comment:
This will need to be removed as it doesn't make sense, marked as future task of the PR.